.github/workflows/pull_requests.yml

```
@@ -0,0 +1,28 @@
+# This workflow will do a clean install of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Testing on PR
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x]
```

Let's update this to `16.x`, for newer Node

```suggestion
        node-version: [16.x]
```

---

Yeah sounds good, I wasn't sure what version was being used.